scalableminds/webknossos
# Integrate "What's New" Notifications/Updates

### URL of deployed dev instance (used for testing):
- https://olvy.webknossos.xyz

### Steps to test:
- open dev instance (isDemo is enabled currently)
- click on "What's New" to see a modal with wK news
- publish some new news on olvy (hit me up for credentials)
- reload wk and expect a red dot on the "What's New" button
- open the modal again --> dot should disappear
- reload page
- dot should be gone

### Issues:
- fixes #5652 

------
